doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -14,32 +14,28 @@ Prepare your PCB
 
 First, you need to create your PCB for the nRF54H20 SoC.
 
-We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources.
-However, if you plan to select your own power sources, consider the following limitations:
+We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources, considering the following limitations:
 
 * The DC/DC inductor must be present on the PCB.
-  If the DC/DC power converter is to be used, the BICR value ``POWER.CONFIG.INDUCTOR`` must also be programmed to ``Present`` (``0``).
-
-  .. note::
-     Currently, there is a bug (LIL-9968) causing the nRF54H20 limited sampling SoC to not wake up from the System OFF if DC/DC is enabled.
-     Temporarily, the workaround is to not use the DC/DC power converter.
-
+* the DC/DC power converter must be present on the PCB and enabled.
+  Also, the ``inductor-present;`` node must be configured either in :file:`zephyr/boards/your_board/nrf54h20dk_bicr.dtsi` or where the section ``bicr: bicr@fff87b0`` is present in the devicetree.
 * For the lowest sleep power consumption, use a 32 KHz crystal.
-* The **P9** port (3 V) cannot be used with internal nor external pull-down resistors.
-* For optimal performance, the output impedance of the **P6** and **P7** ports configured in ``IOPORT.DRIVECTRLX.PY`` should match the PCB and external device pin impedance.
-  However, an existing limitation requires this configuration to be 50 ohms.
+* The **P9** port cannot be used with internal nor external pull-down resistors.
+* For optimal performance, the output impedance of the **P6** and **P7** ports should match the PCB and external device pin impedance.
 
 
 Prepare the configuration files for your custom board in the |NCS|
 ******************************************************************
 
+##TODO
+
 The nRF54H20 DK uses multiple board files for its configuration:
 
 * `ARM cores configuration files`_
```

Comment:
These links point to `next` repos, and should be removed.

I believe the copying you are supposed to do now is:
`sdk-zephyr/boards/nordic/nrf54h20dk` -> `sdk-zephyr/boards/<customer>/<product>`